README.md

```diff
@@ -222,6 +224,26 @@ WebServiceClient client = new WebServiceClient.Builder(6, "ABCD567890").build();
 System.out.println(client.insights(request));
 ```
 
+### Report Transaction
+
+```java
+import java.net.InetAddress;
+import java.util.UUID;
+import com.maxmind.minfraud.request.TransactionReport.Tag;
```

Comment:
I'd either make this example a complete class, remove the imports, or split them off. As is, this isn't valid Java code. I left off the imports for the other examples as people usually let their IDEs handle them, but I think there is a decent argument for making the examples directly runnable, if you want to go that route.

Reply:
Thanks. I've removed the imports. If the IDEs will DWIM then that's good enough for me. :)